### PR TITLE
ci: use app instead of personal token

### PR DESCRIPTION
**Pull request by manuel-rw**
<br/>
<div align="center">
  
  <h3>Homarr</h3>
</div>

**Thank you for your contribution. Please ensure that your pull request meets the following pull request:**

- [x] Builds without warnings or errors (``pnpm buid``, autofix with ``pnpm format:fix``)
- [x] Pull request targets ``dev`` branch
- [x] Commits follow the [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] No shorthand variable names are used (eg. ``x``, ``y``, ``i`` or any abbrevation)

### CODE REVIEW

File: .github/workflows/renovate-automatic-approval.yml [hunk 1]
```diff
@@ -6,18 +6,19 @@ on:
...
       - name: Install GitHub CLI
         run: sudo apt-get install -y gh
-
       - name: Approve Renovate PRs
         env:
-          GITHUB_TOKEN: ${{ secrets.RENOVATE_APPROVE_TOKEN }}
+          GITHUB_TOKEN: ${{ steps.obtainToken.outputs.token }}
         run: |
-          for pr in $(gh pr list --author homarr-renovate[bot] --json number --jq .[].number); do
+          for pr in $(gh pr list --author homarr-approvals[bot] --json number --jq .[].number); do
```
Comment (Meierschlumpf) [inline .github/workflows/renovate-automatic-approval.yml:22]: This should stay the same as homarr-renovate is still the creator of the pull request

File: .github/workflows/renovate-automatic-approval.yml [hunk 2]
```diff
@@ -6,18 +6,19 @@ on:
 jobs:
   approve-renovate-prs:
     runs-on: ubuntu-latest
-
     steps:
-      - name: Checkout code
```
Comment (Meierschlumpf) [inline .github/workflows/renovate-automatic-approval.yml:11]: Checkout no longer needed?

**Review by Meierschlumpf — CHANGES_REQUESTED** [review]

File: .github/workflows/renovate-automatic-approval.yml (hunk 2 above)
Comment (manuel-rw) [reply .github/workflows/renovate-automatic-approval.yml:11]: What would we need the checkout for anyway? This uses GH CLI and nothing else

Comment (Meierschlumpf) [reply .github/workflows/renovate-automatic-approval.yml:11]: I thought it would be required that gh knows which repository it should get the pr's from

**Pull request merged**